### PR TITLE
Rename math symbols

### PR DESCRIPTION
**Pull request by kg583**
A number of math symbols could use some adjustments to their names to improve consistency and ease-of-use. Many proposals have been discussed on Discord, though this PR represents only those with moderate consensus. This PR should remain a draft until all the proposals in the backlog have been majority approved or rejected.

- `square` as a variant is reserved for symbols containing squares; `sq` denotes variants that are just "squarer" (e.g. `union.sq`)
- `aleph`, `beth`, and `gimmel` are the new preferred names for `alef`, `bet`, and `gimel`
- `alef`, `bet`, and `gimel` remain as aliases
- `emptyset` is an alias for `nothing`
- `gt.curly` and `lt.curly` are aliases for `succ` and `pred`
- `equiv` replaces `ident` and is an alias for `eq.triple`
- `ident.strict` is `eq.quad`

Many name changes are implemented as aliases for the time being; whether they should remain aliases indefinitely or just until a future release should be determined on a case-by-case basis.

### CODE REVIEW

**Comment by kg583** [comment]
There was a lot of discussion about the "redundancy" of `.stroked` in variants, but `.stroked` is in fact the default variant for everything except arrows (for which `arrow.fat` was discussed as a substitute, but is probably superfluous now). The actual issue lies with the docs, which state `.stroked` as the canonical variant name. A direct fix to the docs generator is warranted for this, though this could occur either in this PR or during the larger rewrite that needs to happen to implement function scopes.

**Comment by MDLC01** [comment]
I just realised that there is a special, left-to-right, character for the Aleph symbol used to denote transfinite cardinals: U+2135&nbsp;&#8501; ALEF SYMBOL.[^1] Similarly, Unicode supports the following characters:
- U+2136&nbsp;&#8502; BET SYMBOL,
- U+2137&nbsp;&#8503; GIMEL SYMBOL, and
- U+2138&nbsp;&#8504; DALET SYMBOL.

Would it be a good idea to add <code><i>x</i>.symbol</code> (or perhaps <code><i>x</i>.sym</code>), where *x* is a letter name, as variables for the symbols mentioned above? Maybe this is too confusing to most people, since the symbols and their corresponding letters are homoglyphs.

[^1]: https://www.unicode.org/charts/PDF/U2100.pdf

File: library/src/symbols/sym.rs [hunk 1]
```diff
@@ -218,10 +218,23 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
...
         '>',
         circle: '⧁',
+        curly: '≻',
+        curly.approx: '⪸',
+        curly.double: '⪼',
+        curly.eq: '≽',
+        curly.eq.not: '⋡',
+        curly.eqq: '⪴',
+        curly.napprox: '⪺',
+        curly.neqq: '⪶',
```
Comment (rpitasky) [inline library/src/symbols/sym.rs:234]: We should also rename neqq, eqq, etc to nequiv, equiv, etc, to match where relevant, in my opinion.

**Comment by laurmaedje** [comment]
Do we want to merge this at some point? :)

**Comment by kg583** [comment]
Yeah, I suppose its been sitting here long enough. There's still Ryan's comment to address, but other than that we can leave future discussion for a future PR.

**Comment by laurmaedje** [comment]
I'm okay with Ryan's suggestion.

**Comment by kg583** [comment]
Each instance of `eqq` as a variant has been renamed to `equiv`, and likewise for `neqq` to `nequiv`. This conforms pretty well with how `approx` is used as a variant: `napprox` exists only as a variant name, with `approx.not` used for the top level. It also maintains the idea that `not` strikes out the _entire_ symbol, while `nequiv` is only the `equiv` portion struck out (compare `tilde.equiv.not` and `tilde.nequiv`.

**Comment by laurmaedje** [comment]
Thank you all!

**Pull request merged**